docs/src/components/navigation/table-of-contents.tsx

```diff
@@ -0,0 +1,33 @@
+import slugify from '@sindresorhus/slugify';
+
+type Heading = {
+  level: number;
+  text: string;
+};
+
+type TableOfContentsProps = {
+  headings: Heading[];
+};
+
+export function TableOfContents({ headings }: TableOfContentsProps) {
+  return (
+    <div className="w-[12rem] sticky top-10 lg:top-32 self-start hidden md:block">
+      <h5 className="text-xs uppercase text-stone-500">On this page</h5>
```

Comment:
Should this be `h2`? Or `h3`, similar to the other nav?

Reply:
- [ ] TOC Heading structures